ast/format_test.go

```diff
@@ -65,6 +65,9 @@ func (ts *testAstFormatSuite) TestAstFormat(c *C) {
 		{`"abc" not regexp '.*bc?'`, `"abc" NOT REGEXP ".*bc?"`},
 		{`-  4`, `-4`},
 		{`- ( - 4 ) `, `-(-4)`},
+		{`a%b`, "`a` % `b`"},
+		{`a%b+6`, "`a` % `b` + 6"},
```

Comment:
Use ``` instead of %

Reply:
Do you mean put "`a` % `b`" in ``? I have tried but I meet another problem: `a` and `b` are already surround by ``, so we can't use `` to surround them.